pkg/sdk/go/users_test.go

```diff
@@ -281,15 +281,15 @@ func TestUsers(t *testing.T) {
 			token:  token,
 			offset: offset,
 			limit:  0,
-			err:    nil,
+			err:    createError(sdk.ErrFailedFetch, http.StatusInternalServerError),
```

Comment:
Errors related to request limit are `http.StatusBadRequest` and `ErrFailedFetch` is probably too generic as error.

Reply:
That is what is generated by the user service application. We would probably need to change that too.

Reply:
Indeed, `ErrLimitSize` is missing here: https://github.com/mainflux/mainflux/blob/master/users/api/transport.go#L288 It should be added for the case returning `w.WriteHeader(http.StatusBadRequest)`. Can you check if there are more errors missing?
About `ErrFailedFetch` maybe no need to change it since it's an SDK error.